Source/Diagnostics/MultiDiagnostics.H

```diff
@@ -19,6 +19,9 @@ public:
     void InitData ();
     /** \brief Called at each iteration. Compute diags and flush. */
     void FilterComputePackFlush (int step, bool force_flush=false);
+    /** \brief Loop over diags in all diags and call their InitializeFieldFunctors.
+               Called when a new partitioning is generated after load-balancing. */
```

Comment:
```suggestion
    /** \brief Loop over diags in all diags and call their InitializeFieldFunctors.
     * Called at initialization (to create the diagnostics functors) or when a new partitioning
     * is generated after load-balancing, because pointers to WarpX multifabs are dangling then.
     */
```

Reply:
We do not call this function during initialization from MultiDiagnostics.H. We do, however, call InitializeFieldFunctors () in Diagnostics.H from InitData(), which is called from multi_diags. Do you think it would be accurate to mention that this function is called at initialization -- because as of now the loop of all diags to initializefieldfunctors is performed only after LB.

Reply:
Oops no you're right my comment was inaccurate.